kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/KieServerRouter.java

```diff
@@ -271,9 +273,13 @@ public void start(String host,
         HttpHandler blockingHandler = new BlockingHandler(pathHandler);
 
         // main server configuration
-        Undertow.Builder undertowBuilder = Undertow.builder()
-                .addHttpListener(port,
-                                 host);
+        Undertow.Builder undertowBuilder = Undertow.builder();
+        
+        
+        
+        if (isHttpEnabled) {
+            undertowBuilder.addHttpListener(port, host);
+        }
```

Comment:
I think we can show warning in case both HTTP and TLS are disabled, as Smart router doesn't expose any port in such case.

Reply:
I would say that in that case we should throw exception, since the server is useless, wdyt?

Reply:
throwing exception is fine for me too